jlm/llvm/Makefile.sub

```diff
@@ -123,6 +124,7 @@ libllvm_HEADERS = \
 	jlm/llvm/ir/operators/Load.hpp \
 	jlm/llvm/ir/operators/MemCpy.hpp \
 	jlm/llvm/ir/operators/MemoryStateOperations.hpp \
+    jlm/llvm/ir/operators/FunctionPointer.hpp \
```

Comment:
Indentation

Reply:
ah there is a mix of tab/spaces in the file :/ fixed this one now, tending to tabs/spaces at a later point